### DISCUSSION TITLE
resolved

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/Fooocus#331] [Announcements]
resolved

**Comment by greatwolf** [comment]
How do you turn on darkmode for Fooocus as seen in the ui screenshots?

**Reply by lllyasviel** [reply]
my bad:

dark fantasy is here:

![image](https://github.com/lllyasviel/Fooocus/assets/19834515/adc28959-a4a2-4a58-941e-df2cf8e78f0d)

I updated

**Reply by greatwolf** [reply]
isn't that referring to the style of the image generated? I'm actually talking about the gradio interface itself. In A1111 dark mode is enabled by passing `--theme dark` to launch.py but it doesn't look like Fooocus has that option. When I use it in colab for example, the ui is all white.

**Reply by lllyasviel** [reply]
http://127.0.0.1:7860/?__theme=light
or
http://127.0.0.1:7860/?__theme=dark

**Reply by greatwolf** [reply]
thanks

**Comment by dicksondickson** [comment]
Fooocus is amazing. Thank you for your work!

**Comment by MoonRide303** [comment]
@lllyasviel More tests of this feature are needed. I am experimenting with it (on vanilla Fooocus v1.0.50), and while some results are nice, some other are far from what I'd expect from given prompt.

Prompt "a dragon", seed 0:
![image](https://github.com/lllyasviel/Fooocus/assets/130458190/6750db80-b037-4126-a032-7f741f7898b3)

Expended prompt from log:
"a dragon lady, black dress with gold embroidery, necklace, elegant, regal, highly detailed, digital painting, artstation, concept art, smooth, sharp focus, illustration, by artgerm, tomasz alen kopera, peter mohrbacher, donato giancola, joseph christian leyendecker, wlop, boris vallejo"

Maybe it's better to rename it to "Prompt Expansion", and make it disabled by default? It's nice to see enhanced variety in the output images, but it feels too chaotic / unpredictable in current shape.

From prompter perspective I wouldn't mind being it enabed by default, but ONLY under following conditions:
1. It cannot mess up main subject ("a dragon" should stay a dragon, not be turned into "a dragon lady").
2. It shouldn't break desired style - if I want to have cinematic shot, then prompt expansion shouldn't try to turn it into digital painting or illustration.

**Reply by MoonRide303** [reply]
Much better in v1.0.52:

Prompt "a dragon", seed 0:
![image](https://github.com/lllyasviel/Fooocus/assets/130458190/853751cd-c25a-400d-9d60-112aadc1d3da)

Expanded prompt from log:
"a dragon.  Bright explosions.  By Ken Fairclough and Dylan Cole, trending on behance, trending on artstation.  PBR, path tracing, octane render, highly detailed"

**Comment by lllyasviel** [comment]
to be continued in https://github.com/lllyasviel/Fooocus/discussions/117#raw
